.changeset/famous-tigers-cough.md

```diff
@@ -0,0 +1,8 @@
+---
+'@sap-ux/fiori-freestyle-writer': minor
+'@sap-ux/ui5-application-writer': minor
+'@sap-ux/fiori-elements-writer': minor
+'@sap-ux/ui5-library-writer': minor
```

Comment:
```suggestion
'@sap-ux/ui5-application-writer': minor
'@sap-ux/ui5-library-writer': minor
```
I think we can reduce the list here as FE and FF writers are just snapshot updates.

Reply:
> I think we can reduce the list here as FE and FF writers are just snapshot updates.

Yes correct - Thanks